package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "beautiful-react-hooks",
-  "version": "5.0.0",
+  "version": "5.0.1",
```

Comment:
I think semantic release will update the version for you, is gonna be 5.1.0 anyway